### PR TITLE
Remove DeviceExplorer code and references

### PR DESCRIPTION
**Pull request by drwill-ms**

### CODE REVIEW

File: vsts/windows_csharp_deviceexplorer.cmd [hunk 1]
```diff
@@ -19,3 +19,4 @@ echo.
```
Comment (abhipsaMisra) [inline vsts/windows_csharp_deviceexplorer.cmd]: Can get rid of this file

File: readme.md [hunk 2]
```diff
@@ -110,13 +119,16 @@ This repository contains [provisioning service client SDK](./provisioning/servic
...
+
+- [Azure IoT Hub documentation][iot-hub-documentation]
+- [Set up IoT Hub](doc/setup_iothub.md) describes how to configure your Azure IoT Hub service.
+- [Manage IoT Hub](doc/manage_iot_hub.md) describes how to provision devices in your Azure IoT Hub service.
+- [Azure Certified for IoT device catalog](https://catalog.azureiotsuite.com/)
+- [Set up your development environment](./doc/devbox_setup.md) to prepare your development environment as well as how to run the samples on Linux, Windows or other platforms.
+- [API reference documentation for .NET](https://docs.microsoft.com/en-us/dotnet/api/overview/azure/devices?view=azure-dotnet)
+- [Get Started with IoT Hub using .NET](https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-csharp-csharp-getstarted)
+
+> Device Explorer is no longer supported. A replacement tool can be found [here](https://github.com/Azure/azure-iot-explorer).
```
Comment (abhipsaMisra) [inline readme.md]: Can add a note that we can pick the old Device Explorer from Releases tab (also Azure IoT Explorer doesn't seem to support the ability to send C2D messages) (??)

File: doc/manage_iot_hub.md [hunk 3]
```diff
@@ -1,42 +1,27 @@
...
 
-Use either of these tools to generate a device-specific connection string that you can copy and paste in the source code of the application running on your device. 
- 
 > Note: While IoT Hub supports multiple authentication schemes for devices, both these tools generate a pre-shared key to use for authentication.
 
-> Note: You must have an IoT hub running in Azure before you can provision your device. The document [Set up IoT Hub][setup-iothub] describes how to set up an IoT hub.
+> Note: You must have an IoT hub running in Azure before you can provision your device. The document [Set up IoT Hub](./setup_iothub.md) describes how to set up an IoT hub.
 
-You can also use both of these tools to monitor the messages that your device sends to an IoT hub and send commands to your devices from IoT Hub.
+You can also some of these tools to monitor the messages that your device sends to an IoT hub and send commands to your devices from IoT Hub.
```
Comment (abhipsaMisra) [inline doc/manage_iot_hub.md]: typo: You can also _use_ ~~some of~~ these tools

**Comment by xmoulin** [comment]
I think this commit brake some links link for this page: https://docs.microsoft.com/fr-fr/azure/iot-hub/iot-hub-mqtt-support. Link to a broken link "https://github.com/Azure/azure-iot-sdk-csharp/blob/master/tools/DeviceExplorer"

**Comment by drwill-ms** [comment]
@xmoulin, indeed it does. Thank you for notifying us. The Device Explorer tool has long since been discontinued, replaced with the Azure IoT Explorer. (https://github.com/Azure/azure-iot-explorer). Are you able to help update this article with the new tool and link?

**Comment by xmoulin** [comment]
I found others pages talking about DeviceExplorer, so may be it should be a new ticket...
Never try azure-iot-explorer but if try, I'll correct links.

**Pull request merged**